ci/jenkins/jobs/macros.yaml

```diff
@@ -122,7 +122,7 @@
           #!/bin/bash
           set -ex
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
-          ./ci/jenkins/test-vmc.sh --cluster-name "${{JOB_NAME}}-${{BUILD_NUMBER}}" --testcase '{conformance_type}' --coverage --codecov-token "${{CODECOV_TOKEN}}" --registry ${{DOCKER_REGISTRY}}
+          ./ci/jenkins/test-vmc.sh --cluster-name "$BUILD_TAG" --testcase '{conformance_type}' --coverage --codecov-token "${{CODECOV_TOKEN}}" --registry ${{DOCKER_REGISTRY}} --username "${{CAPVC_USERNAME}}" --password "${{CAPVC_PASSWORD}}" --k8s-version "v1.20.1"
```

Comment:
So conformance and networkpolicy tests are using 1.20.1, Antrea e2e is using 1.19.1. I'm OK with this.

Reply:
Note that because of the regression in 1.20.1, test execution time will be extended more or less.